custom/enikshay/integrations/nikshay/tests/test_repeaters.py

```diff
@@ -593,3 +598,316 @@ def test_payload_properties(self, _):
             NikshayTreatmentOutcomePayload(None).get_payload(None, episode_case))
         )
         self.assertEqual(payload['Outcome'], '7')
+
+
+class TestNikshayFollowupRepeater(ENikshayLocationStructureMixin, NikshayRepeaterTestBase):
+
+    def setUp(self):
+        super(TestNikshayFollowupRepeater, self).setUp()
+
+        self.repeater = NikshayFollowupRepeater(
+            domain=self.domain,
+            url='case-repeater-url',
+            username='test-user'
+        )
+        self.repeater.white_listed_case_types = ['test']
+        self.repeater.save()
+
+    def test_not_available_for_domain(self):
+        self.assertFalse(NikshayFollowupRepeater.available_for_domain(self.domain))
+
+    @flag_enabled('NIKSHAY_INTEGRATION')
+    def test_available_for_domain(self):
+        self.assertTrue(NikshayFollowupRepeater.available_for_domain(self.domain))
+
+    def test_followup_for_tests(self):
+        self.assertEqual(NikshayFollowupRepeater().followup_for_tests, ['end_of_ip', 'end_of_cp'])
+
+    @run_with_all_backends
+    def test_trigger(self):
+        self.repeat_record_count = 0
+
+        def check_repeat_record_added():
```

Comment:
nit: I think if you called this `assertRepeatRecordAdded` and raised an `AssertionError` it will result in better error messages if this test fails